java/src/main/java/com/logicalclocks/hsfs/engine/VectorServer.java

```diff
@@ -37,6 +38,8 @@
 import java.util.TreeMap;
 import java.util.stream.Collectors;
 
+import static com.logicalclocks.hsfs.metadata.HopsworksClient.getInstance;
```

Comment:
You have HopsworksClient already imported. You can just call `HopsworksClient.getInstance()` instead of importing the function again separately.